website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,543 @@
+   with the new rule name to verify the login configuration.
+
+   For example:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ curl                                               \
+        --request GET                                    \
+        --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+        ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+    ```
+
+    </CodeBlockConfig>
+    <br/>
+    <CodeBlockConfig hideClipboard>
+
+    ```json
+    {
+      "data": {
+        "backup_auth_types": [
+          "ldap",
+          "token"
+        ],
+        "default_auth_type": "userpass",
+        "disable_inheritance": false,
+        "namespace_path": "dev/"
+      },
+    }
+    ```
+
+    </CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+
+## Delete a login rule
+
+<Tabs>
+<Tab heading="CLI" group="cli">
+
+Use `vault delete` with the 
+[`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+endpoint and rule name to delete a login setting for a namespace. 
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/<unique_rule_name>
+```
+
+For example:
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/dev-gui-login
+```
+
+</CodeBlockConfig>
+
+</Tab>
+<Tab heading="GUI" group="gui">
+
+You can delete a login rule in two places from the **UI Login Settings** page.
+
+**From the additional options menu for the customization**:
+
+1. Find the login rule you want to delete.
+1. Click the additional option button (three dots).
+1. Select **Delete** from the dropdown menu to bring up the delete confirmation modal.
+
+**From the customization details page**:
+
+1. Find the login rule you want to delete.
+1. Select it to open the login settings details page.
+1. Click the **Delete Rule** button to bring up the delete confirmation modal.
+
+</Tab>
+<Tab heading="API" group="api">
+
+Make a `DELETE` call to the
+[`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+endpoint to delete a login setting for a namespace. 
+
+```shell-session
+$ curl                                             \
+  --request DELETE                                 \
+  --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+  http://127.0.0.1:8200/v1/sys/config/ui/login/default-auth/<unique_rule_name>| jq
+```
+
+For example, to delete the rule "dev-gui-login":
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ curl                                               \
+    --request DELETE                                 \
+    --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+    ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+```
+
+</CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+## Using a Direct Link
+
+The `?with=` query parameter can be used at any point to override login customizations. For example,
```

Comment:
@deidraprado I'm tagging you because you’ll have the most insight into how users might interpret this. These are the docs for the custom login feature, and this specific part is about how you can override the default or backup method at any time if you don’t have something like “token” set as one. You and I had discussed this concern, so I wanted your take: any feedback on how this is worded, or do you think it’s clear enough for folks to understand?

Reply:
Sounds good! Most people don't know it's a thing which is why I sprinkled links all over the place 😂 Easy enough to add here, too!